test/test_gnss.py

```diff
@@ -59,10 +57,10 @@ def test_addDateTimeToFiles1(tmp_path, temp_file):
     df = temp_file
 
     with pushd(tmp_path):
-        new_name = os.path.join(tmp_path, 'tmp.csv')
-        df.to_csv(new_name, index=False)
-        addDateTimeToFiles([new_name])
-        df = pd.read_csv(new_name)
+        new_path = tmp_path / 'tmp.csv'
```

Comment:
Is this compatible with running on Win11?

Reply:
Meaning the "/" symbol. I think windows and *nix systems have opposite conventions for which slash they use as directory delimiters.

Reply:
There's a number of other locations where this happens too, so same comment on those.

Reply:
Yes, this is compatible with all platforms. This is actually the Python division operator, which pathlib.Path overloads as its way of joining Paths/strings together. On Linux/Mac/other Unix-based platforms, the resulting path will have forward slashes ("/"), and on Windows it will have backslashes ("\\").